testsuite/libra-swarm/src/main.rs

```diff
@@ -103,8 +103,12 @@ fn main() {
         .config_files
         .iter()
         .map(|config| {
-            let port = NodeConfig::load(config).unwrap().rpc.address.port();
-            format!("localhost:{}", port)
+            let validator_config = NodeConfig::load(config).unwrap();
+            let port = validator_config.rpc.address.port();
+            let debug_interface_port = validator_config
+                .debug_interface
+                .admission_control_node_debug_port;
+            format!("localhost:{}:{}", port, debug_interface_port)
```

Comment:
Can we actually print two separate options for cluster test here?

```
To run transaction generator run: 
(old cmd line without debug port in peers, make sure it works)
To run health check:
(new command line with debug port and --health-check action)

Reply:
Not sure if this is ok. I could not find a way to reuse node_address_list and provide readability.

Reply:
@changyonghu I think in terms of 'how to make it look good', I would separate 'data extraction' and 'data formatting'.

1) Have a var that would hold `Vec<(rpc_port, debug_port)>`.
2) Separately `.map` this var to either 'localhost:rpc_port` or `localhost:rpc_port:debug_port`